docs/deployment/docker.rst

```diff
@@ -10,3 +10,12 @@ Then use Docker to build the image:
 .. code-block:: bash
 
  $ docker build -t mystorefront .
+
+
+Then you can run saleor container with the minimal settings:
```

Comment:
Sounds like we are missing a word at the end of this sentence. Also, `minimal` sounds out of context, maybe we could come up with a better synonym?
```suggestion
Then you can run saleor container with the following fundamental settings:
```

Fundamental or basic?

Reply:
I think just "[...] the following settings [...]" sounds good.